x-pack/plugins/reporting/server/config/schema.ts

```diff
@@ -101,6 +101,45 @@ const PollSchema = schema.object({
   }),
 });
 
+const ExportTypeSchema = schema.object({
+  // Csv reports are enabled in all offerings
+  csv: schema.object({ enabled: schema.boolean({ defaultValue: true }) }),
+  // Png reports are disabled in serverless
+  png: schema.object({
+    enabled: schema.conditional(
+      schema.contextRef('serverless'),
+      true,
+      schema.boolean({
+        validate: (rawValue) => {
+          if (rawValue === true) {
+            return 'PNG reports can only be disabled in serverless mode';
```

Comment:
If I'm reading this right, this message is shown when the config enables serverless and tries to enable the PNG type. Maybe the message should be: `PNG reports can only be enabled in non-serverless mode.`